CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to the Redot Game Engine Website
+
+Thank you for your interest in contributing to the "Redot" Game Engine Website! 🎉  
+This project is an open-source initiative, and we welcome contributions that help improve the website's development in **Vue**, **JavaScript**, **CSS**, **TypeScript**, and **SCSS**.
+
+By contributing, you help create a better user experience for the Redot community.
+
+## How to Contribute
+
+We value any kind of contribution, including (but not limited to):
+
+- Reporting bugs 🐛
+- Suggesting new features or improvements 🚀
+- Fixing bugs 🛠️
+- Improving documentation 📚
+- Refactoring or optimizing code ⚡
+- Writing tests 🧪
+
+## Getting Started
+
+1. **Fork the repository**:  
+   You will need to fork this repository to your own GitHub account, then clone it to your local machine.
+
+2. **Install dependencies**:  
+   After cloning the repository, navigate to the project folder and run:
+
+    ```bash
+    # npm
+    npm install
+    ```
+
+3. Start the development server:
+Run the following command to start a local development environment:
+    ```bash
+    # npm
+    npm run dev
+    ```
+    
+    This will start a server at http://localhost:3000, where you can preview your changes live.
+
+4. Create a new branch:
+When making changes, create a new branch to work on your contribution:
+    ```bash
+    git checkout -b your-feature-branch
+    ```
+5. Make your changes:
+Edit the code, write tests if necessary, and ensure the project is still functioning as expected.
```

Comment:
We don't have tests at this stage but do we need to consider that for this project @YanLi-FE ?